src/pm/hydra/tools/topo/hwloc/topo_hwloc.c

```diff
@@ -139,24 +139,43 @@ static HYD_status split_count_field(const char *str, char **split_str, int *coun
     return status;
 }
 
-static int parse_cache_string(const char *str)
+static void parse_cache_string(const char *str, int *depth, hwloc_obj_cache_type_t * cache_type)
```

Comment:
Do we need a `parse_cache_string` function at all?  Can't we create a simple of options like we do in `comm_split_type`?

Reply:
Yes, will update this. Thank you.